File: JAVA/JavaAssignment/src/main/Ques6.java
package main;

public class Ques6 {
    //There is an array with every element repeated twice except one. Find that element
    public static void main(String[] args) {
        int[]arr={5,6,5,6,8,9,8};
        System.out.println("The element that occurs once is: "+single(arr)+"");
    }

    private static int single(int[] a) {
        int res=a[0];
        for (int i=1;i<a.length;i++){
            res ^=a[i];
        }

        return res;
    }
}

File: KotlinAssignment1/src/Ques6.kt

fun main() {
    //input used
    var arr: Array<Int> = arrayOf(5, 6, 5, 6, 8, 9, 8)
    var res = 0
    for (ml in arr)
        res = res xor ml
    //using xor operator since z xor z returns 0 and z xor 0 returns z
    print("The element that occurs once is $res")
}